src/risedevtool/src/task/compute_node_service.rs

```diff
@@ -102,6 +102,13 @@ impl Task for ComputeNodeService {
             cmd.env("MALLOC_CONF", conf); // unprefixed for linux
         }
 
+        if crate::util::is_env_set("ENABLE_BUILD_RW_CONNECTOR") {
+            cmd.env(
+                "CONNECTOR_LIBS_PATH",
+                ".risingwave/bin/connector-node/libs/",
```

Comment:
What about using env var `PREFIX_BIN`? Not a big deal, but make it more consistent.